charts/newrelic-infrastructure-beta/templates/control-plane/daemonset.yaml

```diff
@@ -0,0 +1,211 @@
+{{- if and (.Values.controlPlane.enabled) (not (include "newrelic.fargate" .)) }}
+apiVersion: apps/v1
+kind: {{ .Values.controlPlane.kind }}
+metadata:
+  namespace: {{ .Release.Namespace }}
+  labels: {{ include "newrelic.labels" . | nindent 4 }}
+  name: {{ template "newrelic.fullname" . }}-control-plane
+  {{- with .Values.controlPlane.annotations }}
+  annotations: {{ toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  {{- with .Values.updateStrategy }}
+  updateStrategy: {{ toYaml . | nindent 4 }}
+  {{- end }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "newrelic.name" . }}-control-plane
+      app.kubernetes.io/instance: {{ .Release.Name }}-control-plane
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        app.kubernetes.io/name: {{ include "newrelic.name" . }}-control-plane
+        app.kubernetes.io/instance: {{ .Release.Name }}-control-plane
+        {{- with .Values.podLabels }}
+          {{- toYaml . | nindent 8 }}
+        {{- end }}
+        mode: {{ include "newrelic.mode" . }}
+    spec:
+      {{- with .Values.images.pullSecrets }}
+      imagePullSecrets:
+          {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ template "newrelic.serviceAccountName" . }}-control-plane
+      {{- if .Values.privileged }}
+      hostNetwork: true
+      dnsPolicy: ClusterFirstWithHostNet
+      {{- end }}
+      containers:
+        - name: control-plane
+          image: {{ .Values.images.scraper.repository }}:{{ .Values.images.scraper.tag | default .Chart.AppVersion }}
+          imagePullPolicy: {{ .Values.images.scraper.pullPolicy }}
+          securityContext:
```

Comment:
All the `securityContext`should be in the values to be idiomatic with the `helm` standards. Take a look to a freshly made `helm create newrelic` in a temporary folder and take it as an example.